<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetParamDlg</class>
 <widget class="QWidget" name="SetParamDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>土壤水分观测仪设备控制</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>F:/MyCPPProject/Image/png/Weather.ico</normaloff>F:/MyCPPProject/Image/png/Weather.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(77,77, 77);color:white</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>46</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QScrollArea" name="ScrollArea_MinBtn">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidgetContents_MinBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="MinBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QScrollArea" name="ScrollArea_CloseBtn">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>3</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidgetContents_CloseBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>18</height>
       </rect>
      </property>
      <widget class="QPushButton" name="CloseBtn">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>1</y>
         <width>18</width>
         <height>18</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="TiltleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../MyCPPProject/Win32/Image/png/control.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel_Name">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>15</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>土壤水分观测仪</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_down">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>数据补抄</string>
     </property>
     <widget class="QDateTimeEdit" name="BdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="EdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="DownBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>补抄数据</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>设备时间</string>
     </property>
     <widget class="QDateTimeEdit" name="RdateTimeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="RTimeBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>读取时钟</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WTimeBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>设置时钟</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_IP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="title">
      <string>IP端口</string>
     </property>
     <widget class="QPushButton" name="RIPBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>读取IP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WIPBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>设置IP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DevicelineEdit_IP1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="DevicelineEdit_IP2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="DevicelineEdit_IP3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="DevicelineEdit_IP4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>25</y>
        <width>10</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>118</x>
        <y>25</y>
        <width>10</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>177</x>
        <y>25</y>
        <width>10</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DevicelineEdit_Port">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>25</y>
        <width>10</width>
        <height>12</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string>：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
